docs/source/getting_started.rst

```diff
@@ -4,42 +4,72 @@
 Getting Started
 ===============
 
-
----------------
-Prerequisites
----------------
+------------
+Dependencies
+------------
 
 PyNWB has the following minimum requirements, which must be installed before you can get started using PyNWB.
 
-#. Python 3.x
-
-   * Before moving forward, make sure `setuptools <https://pypi.python.org/pypi/setuptools>`_ is installed with your version of Python.
+#. Python 2.7 or Python 3.6
+#. pip
+#. Required Packages
 
-#. HDF5
-#. MPI
+   * numpy
+   * h5py
+   * ruamel.yaml
+   * python-dateutil
+   * six
+   * requests
 
 
 The NWB format provides a formal specification for storing neurophysiology data in HDF5. HDF5 provides support
 for parallel I/O using MPI-IO, and therefore requires MPI.
 
 ---------------
 Installation
----------------
+------------
+
+Install from pypi
+-----------------
+
+To install PyNWB distribution from PyPI simply run:
+
+.. code::
+
+   $ pip install pynwb
+
+Install from Git repository
+---------------------------
+
+For development an editable install should be preferred.
```

Comment:
For development an editable install is recommended.

Reply:
May be we should move the Install from git, to the how to contribute section ...

Reply:
I looked at how we did it in Girder and they are in the same spot (which I like). If you think this is a bad idea we can change it though.

Reply:
I think getting started should focus on the goal and how to get there as efficiently as possible:  Read and process NWB files...

This means two code blocks:

* `pip install pynwb`

* powerful and small example to use the library:

```
import pynwb
pynwb. ... 
```

All the possible way of installing should be listed in a "Installation of PyNWB" section.


@nicain  @oruebel : Do you have an example of small snippet that would showcase the power of pynwb ?  

for example, I like how this library is presented: http://requests.readthedocs.io/en/master/